src/components/navbars/DashboardNavTeacher.jsx

```diff
@@ -0,0 +1,40 @@
+import Classes from '../../assets/icons/classes.png';
```

Comment:
Stretch Goal: `DashboardNavStudent` & `DashBoardNavTeacher` are very similar. Could they benefit from the creation of a more abstract component that could handle the different data sets for teachers and students?

Reply:
Got it. I'll think how to do this.